### DISCUSSION TITLE
solito-nativewind-firebase-trpc-prisma

### DISCUSSION THREAD

**Discussion by ebg1223** [nandorojo/solito#185] [Show and tell]
This was created to determine if a client project could be best implemented with this stack:

solito + nativewind + firebase(web + native) + tRPC + prisma starter repo
https://github.com/ebg1223/solito-nativewind-firebase-trpc-prisma-base

might be a little rough, and may not be the best fit for this project, so no promises of maintenance, but in case someone finds this useful.

**Comment by chen-rn** [comment]
Was looking for something exactly like this!
There's also [this](https://github.com/chamatt/create-kaol-app) template that incorporates trpc/solito, inspired by the t3 stack.
